emails/mpa-feedback.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  Body,
+  Column,
+  Container,
+  Head,
+  Heading,
+  Html,
+  Img,
+  Link,
+  Text,
+  Row,
+} from "@react-email/components";
+import { Markdown } from "@react-email/markdown";
+import * as React from "react";
+import StatusImg from "../components/StatusImg";
+import { cdnImageAddress } from "../utils/constants";
+import { StatusImgVariants } from "../utils/enums";
+import { mpaTexts as texts } from "../utils/translations";
+
+export const MpaEmailValidationEmail = () => {
+  return (
+    <Html>
+      <Head />
+      <Body style={main}>
+        <Container style={container}>
+          <Row style={{ ...border, paddingTop: "0" }}>
+            <Column>
+              <Img
+                height={40}
+                src={`${cdnImageAddress}/bratislava-castle-medium.png`}
+                alt="Mesto Bratislava"
+                style={logoImg}
+              />
+            </Column>
+          </Row>
+          <Row style={containerMain}>
+            <Column>
+              <StatusImg variant={StatusImgVariants.sent} />
+              <Heading as="h1" style={{ ...containerMain, ...heading }}>
+                {texts.feedback.title}
+              </Heading>
```

Comment:
Recurring logo and title in every file, better to separate this in new component.